braid/braid.h

```diff
@@ -215,6 +215,16 @@ typedef braid_Int
                      braid_AccessStatus  status            /**< can be querried for info like the current XBraid Iteration */
                      );
 
```

Comment:
Maybe update the comment a bit to

* The sync routine gives the user access to XBraid and to the user's app 
* at various points (primarily once per iteration inside FRefine and outside
*  in the main cycle loop).  This function is called once per-processor (not for every 
* state vector stored on that processor, like access).

Reply:
Changed in https://github.com/XBraid/xbraid/pull/27/commits/bd1d1607c288d5e1d073076837313d33585d079c